Documentation/ceph-upgrade.md

```diff
@@ -294,25 +286,12 @@ This cluster is finished:
   rook-version=v1.2.0
 ```
 
-### 3. Verify the updated cluster
+### 4. Verify the updated cluster
 
 At this point, your Rook operator should be running version `rook/ceph:v1.2.0`.
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
-### 4. Update Rook-Ceph custom resource definitions
-
-> **IMPORTANT**: Do not perform this step until ALL existing Rook-Ceph clusters are updated!
-
-After all Rook-Ceph clusters have been updated following the steps above, update the Rook-Ceph
-Custom Resource Definitions. This is important to get the latest schema validations that will
```

Comment:
@BlaineEXE I don't see the schema validation changes as risky in this release, so updating them before the operator upgrade seems reasonable. Or if you do see a need for the validations to be updated after, we'll need to split the CephClient CRD from the schema updates to avoid errors during the upgrade.

Reply:
I was thinking that it makes sense to keep the CRD updates at the same place in the guide for all releases. I think I'm leaning toward leaving it.

Though the note on external mode isn't necessary now.

Reply:
Ok, to be clear you're suggesting we keep this section? So I'll put this section back and split out the new CRD to create before the upgrade.

Reply:
Yeah I lean toward keeping as is.